src/Bridge/Doctrine/Orm/Util/QueryChecker.php

```diff
@@ -142,12 +142,35 @@ public static function hasOrderByOnToManyJoin(QueryBuilder $queryBuilder, Manage
 
                     if (isset($orderByAliases[$alias])) {
                         $relationship = QueryJoinParser::getJoinRelationship($join);
-                        list($parentAlias, $association) = explode('.', $relationship);
 
-                        $parentMetadata = QueryJoinParser::getClassMetadataFromJoinAlias($parentAlias, $queryBuilder, $managerRegistry);
+                        if (!class_exists($relationship)) {
```

Comment:
hmm not sure strpos is sufficient because `getJoinRelationship` returns the association alone:

```
        $join = new Join('INNER_JOIN', 'relatedDummy', 'a_1', null, 'a_1.name = r.name');
        $this->assertEquals('relatedDummy', QueryJoinParser::getJoinRelationship($join));
```